### ISSUE TITLE
Textfield become read only all the sudden and can't edit any text

### ISSUE THREAD

**Issue by ibm777p2** [FlutterFlow/flutterflow-issues#4910]
### Can we access your project?

- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

When tapping on a textfield that cannot be edited. This issue occurred after duplicating the current project. The duplication caused some textfields to block text input and, in some cases, save the text typed into other textfields in the project. Both the original project and its copies have shown the same error after duplicating the production project.

### Expected Behavior

The expected behavior is that all textfields should allow text input and save the entered text only within their respective fields, without affecting other textfields in the project.

### Steps to Reproduce

1. Clone a project.
2. Create new text field
3. all text field can't be edited and become read only after tapping into it.

### Reproducible from Blank

- [x] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

IT8kz+fqwItIpbxK+KXpbcAwpDgWHFwkRY0/m+0bcAwjCIysEol2d+DsX1tBYsyBeHxYfWeLknsKw+DUhM71C/oBEwqVQYRx3JVqFgvxRjuiVIy4C6WafnRCJ8NXCm7E3MORpAlsA+FaLloX3zmUe+e8VivZHozEOlg7HeaHJZLIjXi5GiPBKidG824XEx7eTI5ODEz25eKMrop5yfj/0w==

### Visual documentation

Here is the code it generate in test mode when the textfield become read only and any text can't be edited.
The password field below I was never able to edit the text after tapping once.
![Imagen de WhatsApp 2024-11-15 a las 16 43 07_301b5af7](https://github.com/user-attachments/assets/746e2b74-142d-4c9b-b5d0-8d31804450a1)
![Imagen de WhatsApp 2024-11-15 a las 18 16 12_e6aaee67](https://github.com/user-attachments/assets/437cfe80-1e18-41d6-b0e6-791f303c5c9b)

### Environment

```markdown
- FlutterFlow version: 3.24.2
- Platform:mobile
- Browser name and version:Chrome 107.0.5304.110
- Operating system and version affected: macOS Sonoma 14.1 (23B2073)
```

**Comment by paulperez-dev** [comment]
Hi @ibm777p2! I wasn’t able to replicate the issue you reported. Could you please check if the issue also occurs in a blank project?

**Comment by Alezanello** [comment]
Closing this issue since we haven't heard back from you. If you are still facing the problem please submit a new issue. Have a great day!

**Comment by ibm777p2** [comment]
> Hi @ibm777p2! I wasn’t able to replicate the issue you reported. Could you please check if the issue also occurs in a blank project?

I haven't seen it in blank project only in the existing one after cloning
